app/networkapi/opportunities/serializers.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework import serializers
+
+from networkapi.opportunities.models import Opportunity
+
+
+class OpportunitySerializer(serializers.ModelSerializer):
+    """
+    Serializes an Opportunity Model
+    """
+    link = serializers.SerializerMethodField('get_link_info')
```

Comment:
let's just use the default value, i.e. no need to specify an arg for this and just rename your method below to `get_link`